rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+format_code_in_doc_comments = true
+format_macro_bodies = true
+format_macro_matchers = true
+group_imports = "StdExternalCrate"
+imports_granularity = "Crate"
+wrap_comments = true
```

Comment:
`rustfmt` seems to have been doing a thing where when I comment out a _block of code_ it decides to "wrap" those lines (so destroying the code's original formatting) which isn't the cutest